lisa/schema.py

```diff
@@ -696,6 +699,17 @@ def __post_init__(self, *args: Any, **kwargs: Any) -> None:
                     "one of admin_password and admin_private_key_file must be set"
                 )
 
+        allow_list: List[Optional[Union[str, bool]]] = [
+            name for name, member in ReserveEnvStatus.__members__.items()
```

Comment:
The allowed list should have "no" also, or user may be confused with this message. And the iterator should be ReserveEnvStatus.__members__.keys(), as the member is not used here.